package.json

```diff
@@ -24,7 +24,9 @@
   "bin": {
     "shjs": "./bin/shjs"
   },
-  "dependencies": {},
+  "dependencies": {
+    "glob": "^6.0.3"
+  },
   "devDependencies": {
```

Comment:
I agree with the philosophy of using modules where they make sense. I think we probably should take advantage of the `glob` module, since it seems to integrate fairly seemlessly with the existing code.